docs/guides/1.18.0-upgrade-guide.md

```diff
@@ -0,0 +1,110 @@
+---
+page_title: "Upgrade Guide 1.18.0"
+---
+
+# MongoDB Atlas Provider 1.18.0: Upgrade and Information Guide
+
+***WARNING:*** For users using the `mongodbatlas_advanced_cluster` resource or data sources, and defining multi sharded clusters, this new version impacts the possibility of editing the definition of these clusters from the Atlas UI. This impact is limited to the first weeks of August 2024.
```

Comment:
`first weeks of August 2024` is a bit vague but I guess it's intentional?

Reply:
Correct we don't have precise timelines from AD3 on when it will be supported apart from what is mentioned.